week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -14,9 +14,19 @@
   6. What value does the "a" parameter take in the first "f1" call?
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
+/*This program throws an error because "b" is not defined as a function argument before
+ So it shoul be defined before execution.
+ Console is printing value of x, a, b 
```

Comment:
I think what 2. asks for is the actual strings that are printed to the console 😉